hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -312,7 +312,6 @@ coverFee_ pparams systemStart epochInfo lookupUTxO walletUTxO partialTx@Babbage.
       costingTx =
         unbalancedTx
           & bodyTxL . outputsTxBodyL %~ (|> feeTxOut)
-          & bodyTxL . feeTxBodyL .~ Coin 10_000_000
```

Comment:
Must: not remove this.

This is needed because the fee is computed based on the serialized size of the body. Having a reasonable (correct order of magnitude) value here, will ensure that we estimate an upper bound.

However, there might be other reasons for why the fee ends up to be too small:
* The `costingTx` does use `feeTxOut` as a stand-in change output, which might be smaller than what we add later with `change`
* The `estimateMinFeeTx` is taking `Babbage` protocol parameters and consequently will not use the new `Conway` specific computation. As we are spending from reference scripts, that is the most likely source for too small fees